test/functional/conftest.py

```diff
@@ -258,9 +269,67 @@ def supports_lxc():
 
 @pytest.helpers.register
 def supports_lxd():
-    return get_lxd_executable()
+    return (get_lxd_executable() and
+            not (os.getenv('TRAVIS') and os.getenv('CI'))  # FIXME: Travis CI
```

Comment:
@gundalow things like this will have to be addressed separately because I think we don't really want this much tests skipped...